internal/pkg/fleetapi/checkin_cmd.go

```diff
@@ -23,6 +23,7 @@ const checkingPath = "/api/fleet/agents/%s/checkin"
 // CheckinRequest consists of multiple events reported to fleet ui.
 type CheckinRequest struct {
 	Status   string        `json:"status"`
+	Message  string        `json:"message"`
```

Comment:
we should add omitempty as well, if no message is provided we can save some bytes

Reply:
added